### PR TITLE
Roles and groups now match service names, only 1 playbook, moved sample inventories into new directory

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

There was duplicating code with use of many all.yml playbooks, slimmed down to 1 playbook file
Moved sample inventories to new directory
Renamed roles, groups, and variables to match the names of the services that get installed
Inventories will need to be updated to match new variables and group names

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

Ran new all.yml to install plaintext, kerberos ssl, and ssl confluent platforms

**Test Configuration**:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.kafka/handlers/main.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
+- name: reload systemd
+  command: systemctl daemon-reload
 - name: restart kafka
   systemd:
-    name: "{{kafka.broker.service_name}}"
+    name: "{{kafka.service_name}}"
```
Comment (JumaX) [inline roles/confluent.kafka/handlers/main.yml]: Recommend changing this to either 'broker' or 'kafka_broker'.

File: roles/confluent.kafka/tasks/main.yml [hunk 2]
```diff
@@ -1,139 +1,139 @@
 - name: broker group
   group:
-    name: "{{kafka.broker.group}}"
+    name: "{{kafka.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend changing this to either 'broker' or 'kafka_broker'.

File: roles/confluent.kafka/tasks/main.yml [hunk 3]
```diff
@@ -1,139 +1,139 @@
...
     comment: "Kafka User"
     system: yes
-    group: "{{kafka.broker.group}}"
+    group: "{{kafka.group}}"
 - name: Change keytabs ownership
   file:
-    path: "{{kafka.broker.keytab}}"
-    group: "{{kafka.broker.group}}"
+    path: "{{kerberos.keytab_dir}}"
+    group: "{{kafka.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 4]
```diff
@@ -1,139 +1,139 @@
...
     mode: '650'
     recurse: yes
 - name: create broker data directories
   file:
     path: "{{item}}"
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
+    owner: "{{kafka.user}}"
+    group: "{{kafka.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 5]
```diff
@@ -1,139 +1,139 @@
...
     state: directory
     mode: 0755
-  with_items: "{{kafka.broker.datadir}}"
+  with_items: "{{kafka.datadir}}"
 - name: broker plaintext config
   template:
     src: server.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 6]
```diff
@@ -1,139 +1,139 @@
...
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
+    owner: "{{kafka.user}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 7]
```diff
@@ -1,139 +1,139 @@
...
+    group: "{{kafka.group}}"
   when: security_mode == "plaintext"
   notify:
     - restart kafka
 - name: broker ssl config
   template:
     src: server_ssl.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 8]
```diff
@@ -1,139 +1,139 @@
...
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 9]
```diff
@@ -1,139 +1,139 @@
...
+    owner: "{{kafka.user}}"
+    group: "{{kafka.group}}"
   when: security_mode == "ssl"
   notify:
     - restart kafka
 - name: broker sasl_ssl config
   template:
     src: server_sasl_ssl.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 10]
```diff
@@ -1,139 +1,139 @@
...
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 11]
```diff
@@ -1,139 +1,139 @@
...
+    owner: "{{kafka.user}}"
+    group: "{{kafka.group}}"
   when: security_mode == "sasl_ssl"
   notify:
     - restart kafka
 - name: broker ssl custom signed config
   template:
     src: server_ssl_self_signed.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 12]
```diff
@@ -1,139 +1,139 @@
...
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
+    owner: "{{kafka.user}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 13]
```diff
@@ -1,139 +1,139 @@
...
     - restart kafka
 - name: broker gssapi kerberos config
   template:
     src: server_kerberos.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
+    owner: "{{kafka.user}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 14]
```diff
@@ -1,139 +1,139 @@
...
   notify:
     - restart kafka
 - name: broker ssl kerberos config
   template:
     src: server_kerberos_ssl.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 15]
```diff
@@ -1,139 +1,139 @@
...
   notify:
     - restart kafka
 - name: broker ssl kerberos config
   template:
     src: server_kerberos_ssl.self_signed.properties.j2
-    dest: "{{kafka.broker.config_file}}"
+    dest: "{{kafka.config_file}}"
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 16]
```diff
@@ -1,139 +1,139 @@
...
 - name: broker Jaas config
   template:
     src: kafka_server_jaas.conf.j2
-    dest: "{{kafka.broker.jaas_file}}"
+    dest: "{{kafka.jaas_file}}"
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
+    owner: "{{kafka.user}}"
+    group: "{{kafka.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 17]
```diff
@@ -1,139 +1,139 @@
...
   notify:
     - restart kafka
 - name: create systemd override file
   file:
-    path: "{{kafka.broker.systemd_override}}"
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
+    path: "{{kafka.systemd_override}}"
+    owner: "{{kafka.user}}"
+    group: "{{kafka.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 18]
```diff
@@ -1,139 +1,139 @@
...
     state: directory
     mode: 0640
 - name: write environment variables
   template:
     src: environment.j2
-    dest: "{{kafka.broker.systemd_override}}/override.conf"
+    dest: "{{kafka.systemd_override}}/override.conf"
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 19]
```diff
@@ -1,139 +1,139 @@
...
     - reload systemd
     - restart kafka
 - name: write environment variables
   template:
     src: kerberos_environment.j2
-    dest: "{{kafka.broker.systemd_override}}/override.conf"
+    dest: "{{kafka.systemd_override}}/override.conf"
     mode: 0640
-    owner: "{{kafka.broker.user}}"
-    group: "{{kafka.broker.group}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/tasks/main.yml [hunk 20]
```diff
@@ -1,139 +1,139 @@
...
   when: security_mode == "kerberos" or security_mode == "kerberos_ssl" or security_mode == "kerberos_ssl_customcerts"
   notify:
     - reload systemd
     - restart kafka
 - name: broker configure service
   systemd:
-    name: "{{kafka.broker.service_name}}"
-    enabled: "{{kafka.broker.systemd.enabled}}"
-    state: "{{kafka.broker.systemd.state}}"
+    name: "{{kafka.service_name}}"
```
Comment (JumaX) [inline roles/confluent.kafka/tasks/main.yml]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server.properties.j2 [hunk 21]
```diff
@@ -1,7 +1,7 @@
 # Maintained by Ansible
-listeners=PLAINTEXT://:{{broker.config.port}}
+listeners=PLAINTEXT://:{{kafka_port}}
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_kerberos.properties.j2 [hunk 22]
```diff
@@ -4,8 +4,8 @@
 
 sasl.enabled.mechanisms=GSSAPI
 sasl.mechanism.inter.broker.protocol=GSSAPI
-listeners=SASL_PLAINTEXT://{{inventory_hostname}}:{{broker.config.sasl_port}}
-advertised.listeners=SASL_PLAINTEXT://{{inventory_hostname}}:{{broker.config.sasl_port}}
+listeners=SASL_PLAINTEXT://{{inventory_hostname}}:{{kafka_port}}
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_kerberos.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_kerberos_ssl.properties.j2 [hunk 23]
```diff
@@ -1,7 +1,7 @@
 # Maintained by Ansible
-listeners=SASL_SSL://{{inventory_hostname}}:{{broker.config.sasl_port}},PLAINTEXT://{{inventory_hostname}}:9091
+listeners=SASL_SSL://{{inventory_hostname}}:{{kafka_port}},PLAINTEXT://{{inventory_hostname}}:9091
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_kerberos_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_kerberos_ssl.properties.j2 [hunk 24]
```diff
@@ -1,7 +1,7 @@
...
 listener.security.protocol.map=SASL_SSL:SASL_SSL,PLAINTEXT:PLAINTEXT
-advertised.listeners=SASL_SSL://{{inventory_hostname}}:{{broker.config.sasl_port}},PLAINTEXT://{{inventory_hostname}}:9091
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_kerberos_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_kerberos_ssl.self_signed.properties.j2 [hunk 25]
```diff
@@ -2,7 +2,7 @@
 
 {% include './includes/base_server.properties.j2' %}
 
-listeners=SASL_SSL://:{{broker.config.port}}
+listeners=SASL_SSL://:{{kafka_port}}
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_kerberos_ssl.self_signed.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_sasl_ssl.properties.j2 [hunk 26]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-listeners=SASL_SSL://:{{broker.config.port}}
+listeners=SASL_SSL://:{{kafka_port}}
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_sasl_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_ssl.properties.j2 [hunk 27]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-listeners=SSL://:{{broker.config.port}}
+listeners=SSL://:{{kafka_port}}
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka/templates/server_ssl_self_signed.properties.j2 [hunk 28]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-listeners=SSL://:{{broker.config.port}}
+listeners=SSL://:{{kafka_port}}
```
Comment (JumaX) [inline roles/confluent.kafka/templates/server_ssl_self_signed.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka_connect/templates/includes/base_connect-distributed.properties.j2 [hunk 29]
```diff
@@ -1,13 +1,13 @@
 {% set schema_registries = groups.get('schema-registry', []) %}
 
-bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{broker.config.port}}{% endfor %}
+bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.kafka_connect/templates/includes/base_connect-distributed.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka_rest/templates/kafka-rest_kerberos_ssl.properties.j2 [hunk 30]
```diff
@@ -1,16 +1,16 @@
 # Maintained by Ansible
 {% include './includes/base_kafka-rest.properties.j2' %}
 
-client.bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_PLAINTEXT://{{ inventory_hostname }}:{{broker.config.port}}{% endfor %}
+client.bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_PLAINTEXT://{{ inventory_hostname }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.kafka_rest/templates/kafka-rest_kerberos_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.kafka_rest/templates/kafka-rest_kerberos_ssl_customcerts.properties.j2 [hunk 31]
```diff
@@ -1,7 +1,7 @@
 # Maintained by Ansible
 {% include './includes/base_kafka-rest.properties.j2' %}
 
-client.bootstrap.servers={% for inventory_hostname in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{broker.config.port}}{% endfor %}
+client.bootstrap.servers={% for inventory_hostname in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.kafka_rest/templates/kafka-rest_kerberos_ssl_customcerts.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.ksql/templates/includes/base_ksql.properties.j2 [hunk 32]
```diff
@@ -1,6 +1,6 @@
 {% set schema_registries = groups.get('schema-registry', []) %}
 
-bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{broker.config.port}}{% endfor %}
+bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.ksql/templates/includes/base_ksql.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl.properties.j2 [hunk 33]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for inventory_hostname in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for inventory_hostname in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ inventory_hostname }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl_customcerts.properties.j2 [hunk 34]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ host }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl_customcerts.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.schema_registry/templates/schema-registry_sasl_ssl.properties.j2 [hunk 35]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ host }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SASL_SSL://{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.schema_registry/templates/schema-registry_sasl_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.schema_registry/templates/schema-registry_ssl.properties.j2 [hunk 36]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}SSL://{{ host }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SSL://{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.schema_registry/templates/schema-registry_ssl.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.schema_registry/templates/schema-registry_ssl_customcerts.properties.j2 [hunk 37]
```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}SSL://{{ host }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SSL://{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.schema_registry/templates/schema-registry_ssl_customcerts.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

File: roles/confluent.tools/templates/includes/base_tools.properties.j2 [hunk 38]
```diff
@@ -1,4 +1,4 @@
-bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{broker.config.port}}{% endfor %}
+bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_port}}{% endfor %}
```
Comment (JumaX) [inline roles/confluent.tools/templates/includes/base_tools.properties.j2]: Recommend leaving in 'broker' as kafka is not a component but the platform name.

**Comment by JumaX** [comment]
Additional notes:

1. in all.yml, why do we not gather facts?
2. Are we doing a separate PR for exposing the variables?

File: roles/confluent.schema_registry/templates/schema-registry_ssl.properties.j2 (hunk 36 above)
Comment (domenicbove) [reply roles/confluent.schema_registry/templates/schema-registry_ssl.properties.j2]: going with kafka_broker

File: roles/confluent.schema_registry/templates/schema-registry_sasl_ssl.properties.j2 (hunk 35 above)
Comment (domenicbove) [reply roles/confluent.schema_registry/templates/schema-registry_sasl_ssl.properties.j2]: going with kafka_broker

File: roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl_customcerts.properties.j2 (hunk 34 above)
Comment (domenicbove) [reply roles/confluent.schema_registry/templates/schema-registry_kerberos_ssl_customcerts.properties.j2]: going with kafka_broker

File: roles/confluent.kafka/tasks/main.yml (hunk 2 above)
Comment (domenicbove) [reply roles/confluent.kafka/tasks/main.yml]: kafka_broker

**Comment by JumaX** [comment]
@domenicbove new issue, the inventory files are incorrect.  You are using `-` instead of `_` like in the all.yml.

ex. kafka-connect in invetory vs. kafka_connect in all.yml

File: roles/confluent.common/tasks/debian_jvm.yml [hunk 39]
```diff
@@ -21,11 +21,13 @@
 
 - name: Install rsync
   apt:
-    name: rsync
+    name: "{{ item }}"
     state: present
-  with_items:
```
[the remark this answers is not in the document]
Comment (domenicbove) [reply roles/confluent.common/tasks/debian_jvm.yml]: ansible is moving with_items -> loop. this change eliminates a deprecation warning. but its the same functionality

File: roles/confluent.common/tasks/redhat.yml [hunk 40]
```diff
@@ -1,25 +1,25 @@
+- name: Install Java
+  yum:
+    name: "java-1.8.0-openjdk"
```
[the remark this answers is not in the document]
Comment (domenicbove) [reply roles/confluent.common/tasks/redhat.yml]: its in a variable now

**Pull request merged**